introduction.md

```diff
@@ -0,0 +1,5 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris congue, eros et cursus aliquet, mi lectus efficitur massa, id ultrices lectus elit quis justo. Pellentesque lacinia mi non convallis aliquam. Cras consequat urna sed leo hendrerit porta. Donec in nulla lacinia, imperdiet odio in, pharetra elit. Vivamus hendrerit felis in cursus pellentesque. Duis tincidunt sagittis scelerisque. Sed egestas euismod lectus, ac rhoncus lacus commodo a. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Proin sit amet nisl turpis. Fusce a nulla erat. Suspendisse laoreet nisi leo, convallis blandit nunc placerat eget. Proin vitae viverra magna. Aliquam condimentum diam nisl, nec dignissim sem tristique vitae.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.